internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -765,6 +765,16 @@ func (r *AgentServiceConfigReconciler) newImageServiceDeployment(ctx context.Con
 				},
 			},
 		},
+		LivenessProbe: &corev1.Probe{
+			InitialDelaySeconds: 30,
```

Comment:
Why the delay?

Reply:
I took the same as assisted-service:
https://github.com/openshift/assisted-service/blob/master/internal/controller/controllers/agentserviceconfig_controller.go#L915